### ISSUE TITLE
AVR4L for every linux distro 

### ISSUE THREAD

**Issue by cvam0000** [abdalmoniem/AVR4L#19]
We should make AVR4L available for each Linux distro, not only for ubuntu

**Comment by abdalmoniem** [comment]
yeah sure, but this'll need a massive development time and a big team. if you know of anyone who's interested in developing please let them know

**Comment by cvam0000** [comment]
okay i will let you know soon

**Comment by cvam0000** [comment]
ok, let's make an organization and start working @abdalmoniem

**Comment by abdalmoniem** [comment]
sure, lets start by making a slack channel for communication [here](https://avr4lcommunity.slack.com)

**Comment by cvam0000** [comment]
telegram is a good option I guess

**Comment by abdalmoniem** [comment]
ok, not bad

**Comment by abdalmoniem** [comment]
https://t.me/avr4lcommunity

**Comment by cvam0000** [comment]
make a group so, i can add some more developers

**Comment by abdalmoniem** [comment]
anyone can join through this link

**Comment by cvam0000** [comment]
how do i write a message in the group ??

**Comment by cvam0000** [comment]
Im not able to answer on the telegram please make it a group

**Comment by abdalmoniem** [comment]
to make the group I need to add at least one contact, so I need your phone number to add you

**Comment by cvam0000** [comment]
cvam0000 is my username

**Comment by OMGdaDPS** [comment]
flatpak it or snap it up!

**Comment by OMGdaDPS** [comment]
then anything you add to it here, will automatically get pushed and updated in the flatpak!

**Comment by cvam0000** [comment]
what ??
@OMGdaDPS

**Comment by OMGdaDPS** [comment]
I was saying that you could turn your program into a flatpak. You could even just have them do it i think. A flatpak is distro-agnostic, which means that as long as a user has the flatpak stuff installed on their linux system, they will be able to run it!

**Comment by OMGdaDPS** [comment]
and when you make the flatpak, because of the way that you do it, whenever you push an update or make changes to it here, it will automatically get built into the flatpak !
